_templates/collection/new/prompt.js

```diff
@@ -0,0 +1,20 @@
+// see types of prompts:
+// https://github.com/enquirer/enquirer/tree/master/examples
+//
+module.exports = [
+  {
+    type: 'input',
+    name: 'collectionName',
+    message: "Collection name:"
+  },
+  {
+    type: 'input',
+    name: 'collectionVersion',
+    message: "Collection version:"
+  },
+  {
+    type: 'input',
+    name: 'sampleFilenamePrefix',
+    message: "Sample Filename Prefix"
```

Comment:
trivial: for consistency, let's use `"Sample filename prefix:"` (matching casing with the other messages, as well as a trailing colon)

Reply:
Updated